webook/calendar_buddy/contexts/fullcalendar/factory.py

```diff
@@ -0,0 +1,60 @@
+from . import standard_library
+from ....calendar_buddy import base
+from . context import FullCalendarContext
+
+
+class FullCalendarFactory(base.BaseCalendarContextFactory):
+    """
+        A factory which can fabricate FullCalendarContexts
+    """
+
+    def __init__(self, standard_ui_config: base.UIConfig = None, event_standard: dict() = None, resource_standard: dict() = None) -> None:
+        """
+            Initialize the FullCalendarFactory
+
+            :param standard_ui_config: The standard base UI config that this factory is to adhere to when fabricating in the future. May be None.
+            :type standard_ui_config: UIConfig
+
+            :param event_standard: The standard event defaults that this factory is to adhere to when fabricating. May be None.
+            :type event_standard: dict
+
+            :param resource_standard: The standard resource defaults that this factory is to adhere to when fabricating. May be None.
+            :type resource_standard: dict
+        """
+        super().__init__()
+        self.standard_ui_config = base.UIConfig().overwrite(standard_ui_config)
+
+        self.event_standard = standard_library.get_base_event_standard()
+        self.resource_standard = standard_library.get_base_resource_standard()
+
+        # use functionality from base to mesh the base/standard default (default if nothing is supplied), with the
+        # eventual defaults of the user.
+        self.event_standard = self._get_standard_event_default(specified_defaults=event_standard)
+        self.resource_standard = self._get_standard_resource_default(specified_defaults=resource_standard)
+
+    def fabricate(self, custom_event_standard: list() = None, custom_resource_standard: list() = None) -> FullCalendarContext:
```

Comment:
dont run the list function  in a type hint. there are more then one of these. so search and replace them